CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ Release report: TBD
 
 ### Changed
 
+- Update FIM `test_audit` tests to new framework ([#3939](https://github.com/wazuh/wazuh-qa/pull/3939)) \- (Tests)
```

Comment:
```suggestion
- Update FIM `test_audit` tests to new framework ([#3939](https://github.com/wazuh/wazuh-qa/pull/3939)) \- (Framework + Tests)
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/3939/commits/f56f61577f3310474fbdac39dcf3a2f8401f2a8c